radixdlt-core/radixdlt/src/main/java/com/radixdlt/statecomputer/RadixEngineStateComputer.java

```diff
@@ -226,7 +226,7 @@ public void addToMempool(MempoolAdd mempoolAdd, @Nullable BFTNode origin) {
         .forEach(
             txn -> {
               try {
-                addToMempool(txn);
+                addToMempool(txn, origin);
               } catch (MempoolRejectedException ignored) {
               }
```

Comment:
I understand that the exception here is ignored, but perhaps we can add at least logging here (perhaps even at debug level)? I feel uncomfortable if something might happen without at least leaving any signs in logs.

Reply:
sure

Reply:
done